byok/generate_embeddings_tool.py

```diff
@@ -92,7 +95,11 @@ def file_metadata_func(file_path: str) -> Dict:
 
     # Load documents
     documents = SimpleDirectoryReader(
-        args.input_dir, recursive=True, file_metadata=file_metadata_func
+        args.input_dir,
+        recursive=True,
+        required_exts=[".md"],
```

Comment:
why we restrict to markdown files only?

Reply:
Because that's the only format we're prepared to parse.